packages/editor/playground/playground.tsx

```diff
@@ -94,35 +124,50 @@ const App: FC = () => {
     const initialContentId = params.get("contentid")
     const enableDevTools = params.get("devtools") !== "false"
 
-    const note = useMemo(() => {
-        let content = defaultContent
-        if (isString(initialContent)) {
-            content = initialContent
-        } else if (initialContentId === "long") {
-            content = longContent
-        } else if (initialContentId === "just-code") {
-            content = justCodeContent
-        }
+    const [hasUnsavedChanges, setHasUnsavedChanges] = useState(false)
 
+    if (isString(initialContent) && initialContent.length > 0) {
+        contentMap["customize"] = initialContent
```

Comment:
```suggestion
        contentMap["customize"] = initialContent
        initialContentId = "customize"
```

Reply:
I guess that you want to make `initialContent` having priority over `contentId`.
But I'm thinking that this feature is somewhat weird without saying that your suggest changing is not working actually.